controllers/pageCrawler.js

```diff
@@ -24,11 +24,13 @@ async function scrapeURL(productURL, scrapedURLs) {
         .first()
         .text();
       const numPrice = cleanPrice(price) || -1;
+      const description = $('div.product--description').text();
 
-      console.log(title, img, price, numPrice);
+      console.log(title, description, img, price, numPrice);
```

Comment:
console.log?? Do you still need it here?

Reply:
So far yes. I will remove this one in a later stage.